recipes/r-readtext/meta.yaml

```diff
@@ -0,0 +1,103 @@
+{% set version = '0.80' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-readtext
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/readtext_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/readtext/readtext_{{ version }}.tar.gz
+  sha256: be0016339eb081a60e26cbe9dd779695378edff8a2ce7b49809bbb94ae950a61
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
```

Comment:
This should be `noarch: generic` now. Given that the Linux build seems to work, this should be mergable afterwards.

Reply:
@cbrueffer windows has failed now

Reply:
seems like we will need to skip win and r-base 4.0.

Reply:
The `noarch: generic` mentioned above takes care of this by building one package that works on all platforms; those packages are built on Linux, the Windows failure doesn't matter then.

Reply:
And it allows for windows users to install this in an environment with r 3.6?

Reply:
Yes. There'll be one package per R version, should have been more specific.